### ISSUE TITLE
i18n "translated percentage" image

### ISSUE THREAD

**Issue by willow446** [willow446/willow446.github.io#38]
Currently there is a badge that shows the ouisync weblate translation percentage. It is a hardcoded svg path so it will load the same label "translated" for all languages. Not sure if this is possible but would be a nice touch.

cc: @jnny do you have any clue if this is possible?

**Comment by jnny** [comment]
I don't fully understand the question, could you rephrase?
Currently the Community page pulls percent translated to target languages from Ouisync's Weblate (which still needs work..). Some of the documentation on the badges (or "shields"??) can be found here: https://github.com/badges/shields/pull/6677

**Comment by willow446** [comment]
It's fine as is. Feel free to open this up again if there's any interest in doing this.

**Comment by jnny** [comment]
I still don't understand the issue - doing what exactly?
